tests/unit_tests/conftest.py

```diff
@@ -13,6 +14,18 @@
 settings.load_profile("ci")
 
 
+@pytest.fixture(autouse=True)
+def mock_servier_viewer_api_call():
```

Comment:
```suggestion
def mock_server_viewer_api_call():
```

also document why this is here, why it's an autouse fixture, and if anyone might need to be aware of this in tests